pkgs/by-name/ra/rapidraw/package.nix

```
@@ -0,0 +1,148 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  rustPlatform,
+  makeWrapper,
+  pkg-config,
+  wrapGAppsHook4,
+  openssl,
+  onnxruntime,
+  webkitgtk_4_1,
+  gtk3,
+  glib,
+  gdk-pixbuf,
+  libappindicator,
+  cairo,
+  pango,
+  xorg,
+  libxkbcommon,
+  vulkan-loader,
+  libjpeg,
+  libpng,
+  zlib,
+  libGL,
+  dbus,
+  gvfs,
+  libheif,
+  glib-networking,
+  gsettings-desktop-schemas,
+  nodejs_20,
+  npmHooks,
+  cargo-tauri,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rapidraw";
+  version = "1.2.6";
+
+  src = fetchFromGitHub {
+    owner = "CyberTimon";
+    repo = "RapidRAW";
+    rev = "70de8ed1bb77f0d7243c1a79be7c83143d59d085";
```

Use tags, if there are any
```suggestion
    tag = "v${finalAttrs.version}";
```

---

oh sweet! i had originally done it that way but when reading the checklist thought it advised against that